annotation-utils/src/main/java/edu/isi/vista/annotationutils/RestoreJson.kt

```diff
@@ -146,4 +156,20 @@ private fun getAceDocID(filename: String): String? {
         return matchResult!!.groups[1]!!.value
     }
     return null
+}
+
+private fun getCord19DocID(filename: String): String? {
+    // Returns empty string if not a CORD-19 filename
+    // CORD-19 filenames consist of a string of letters and numbers
+    // and then the annotator's username
+    // Example: hfofi34yrohwfniw94tyowlefnweiug8iryfhwen-john_bob.json
+
+    val regex = Regex(pattern = """([a-z0-9]{30,})-.*""")
+    if (regex.containsMatchIn(filename)) {
+        val matchResult = regex.find(filename)
+        //Returns only the doc id. In this case: CNNHL_ENG_20030304_142751.10
+        logger.info { "CORD-19 ID: ${matchResult!!.groups[1]!!.value}" }
+        return matchResult!!.groups[1]!!.value

```

Comment:
nit: Not super familiar with Kotlin, but this seems like an expression that could be made multi-line for clarity